### ISSUE TITLE
[arch] py3status 3.3 (non-git) broken after latest Python update to 3.6

### ISSUE THREAD

**Issue by lasers** [ultrabug/py3status#611]
`aur/py3status 3.3-1 (25, 1.36)` <- BROKEN.
`aur/py3status-git 2.4.r42.gc4535a4-1 (17, 0.13) [installed: 3.3.r72.gd37b9f2-1]` <- OKAY.

I moved to `-git`. Okay now.  I didn't flag it because it's probably not out of date, just broken.

**Comment by tobes** [comment]
I think this is due to arch updating to python 3.6 there was an issue #606 

@ultrabug maybe we should look at releasing py3status v3.4 to help with the python 3.6 transition.

**Comment by tobes** [comment]
@Horgix not sure if you are aware of this issue.  python 3.6 breaks py3status v3.3 the fix is in PR #610

**Comment by raspbeguy** [comment]
I confirm the bug.

**Comment by jonathanio** [comment]
The py3status package in AUR has just been updated to 3.3-2 with a temporary patch to fix this until 3.4 is released.

**Comment by tobes** [comment]
> The py3status package in AUR has just been updated to 3.3-2 with a temporary patch to fix this until 3.4 is released.

great news :)

**Comment by Horgix** [comment]
Wow, since I subscribed to the entire py3status project on Github I received so much notifications by mail that I ended up filtering it to a folder... so I missed that, my bad. Guess I'll change what I'm notified about!

Anyway, as @jonathanio said, it was fixed by [the patch he generously sent to me](https://gist.github.com/jonathanio/fb1058c02b4d1cecc78ee369db401570). See comments https://aur.archlinux.org/packages/py3status/?comments=all for details 🙂 

I think we can close this now, but I'm not fan of patching code in the package, so the fastest it will be shipped in a new release, the better!
